### PR TITLE
Change the support provider to SBO Community

### PR DESCRIPTION
**Pull request by baijum**
### Motivation

The upstream project is supported by SBO Community

### Changes

Change the support entity

### Testing

NA

For further more details refer the CONTRIBUTING.md

### CODE REVIEW

File: config/manifests/bases/service-binding-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
     description: An operator to support binding capabilities between imported apps
       and operator backed services
     repository: https://github.com/redhat-developer/service-binding-operator
-    support: Red Hat
+    support: SBO Community
```
Comment (sbose78) [inline config/manifests/bases/service-binding-operator.clusterserviceversion.yaml:11]: *Service Binding Operator Community

Comment (baijum) [reply config/manifests/bases/service-binding-operator.clusterserviceversion.yaml]: done

**Pull request merged**